### ISSUE TITLE
Uninitialized constant AdditionalsTag::RedmineCrm (Updating to 2.0.16)

### ISSUE THREAD

**Issue by dktcoding** [AlphaNodes/additionals#35]
I get the following issue when attempting to upgrade `Additionals` to version `2.0.16`:

~~~log
[ E 2018-12-11 17:46:49.9667 332/T1dl age/Cor/App/Implementation.cpp:221 ]: Could not spawn process for application /home/redmine/redmine: The application encountered the following error: uninitialized constant AdditionalsTag::RedmineCrm (NameError)
  Error ID: a695f5f8
  Error details saved to: /tmp/passenger-error-QgTOXp.html

[ E 2018-12-11 17:46:49.9735 332/Td age/Cor/Con/CheckoutSession.cpp:276 ]: [Client 3-807] Cannot checkout session because a spawning error occurred. The identifier of the error is a695f5f8. Please see earlier logs for details about the error.
App 26531 output: Error: The application encountered the following error: uninitialized constant AdditionalsTag::RedmineCrm (NameError)
App 26531 output:     /home/redmine/redmine-4.0.0/plugins/additionals/app/models/additionals_tag.rb:2:in `<class:AdditionalsTag>'
App 26531 output:     /home/redmine/redmine-4.0.0/plugins/additionals/app/models/additionals_tag.rb:1:in `<top (required)>'
App 26531 output:     /var/lib/gems/2.5.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
App 26531 output:     /var/lib/gems/2.5.0/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
App 26531 output:     /var/lib/gems/2.5.0/gems/activesupport-5.2.2/lib/active_support/dependencies/interlock.rb:14:in `block in loading'
App 26531 output:     /var/lib/gems/2.5.0/gems/activesupport-5.2.2/lib/active_support/concurrency/share_lock.rb:151:in `exclusive'
App 26531 output:     /var/lib/gems/2.5.0/gems/activesupport-5.2.2/lib/active_support/dependencies/interlock.rb:13:in `loading'
App 26531 output:     /var/lib/gems/2.5.0/gems/railties-5.2.2/lib/rails/engine.rb:478:in `block (2 levels) in eager_load!'
App 26531 output:     /var/lib/gems/2.5.0/gems/railties-5.2.2/lib/rails/engine.rb:477:in `each'
App 26531 output:     /var/lib/gems/2.5.0/gems/railties-5.2.2/lib/rails/engine.rb:477:in `block in eager_load!'
App 26531 output:     /var/lib/gems/2.5.0/gems/railties-5.2.2/lib/rails/engine.rb:475:in `each'
App 26531 output:     /var/lib/gems/2.5.0/gems/railties-5.2.2/lib/rails/engine.rb:475:in `eager_load!'
App 26531 output:     /var/lib/gems/2.5.0/gems/railties-5.2.2/lib/rails/engine.rb:356:in `eager_load!'
App 26531 output:     /var/lib/gems/2.5.0/gems/railties-5.2.2/lib/rails/application/finisher.rb:69:in `each'
App 26531 output:     /var/lib/gems/2.5.0/gems/railties-5.2.2/lib/rails/application/finisher.rb:69:in `block in <module:Finisher>'
App 26531 output:     /var/lib/gems/2.5.0/gems/railties-5.2.2/lib/rails/initializable.rb:32:in `instance_exec'
App 26531 output:     /var/lib/gems/2.5.0/gems/railties-5.2.2/lib/rails/initializable.rb:32:in `run'
App 26531 output:     /var/lib/gems/2.5.0/gems/railties-5.2.2/lib/rails/initializable.rb:61:in `block in run_initializers'
App 26531 output:     /usr/lib/ruby/2.5.0/tsort.rb:228:in `block in tsort_each'
App 26531 output:     /usr/lib/ruby/2.5.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
App 26531 output:     /usr/lib/ruby/2.5.0/tsort.rb:431:in `each_strongly_connected_component_from'
App 26531 output:     /usr/lib/ruby/2.5.0/tsort.rb:349:in `block in each_strongly_connected_component'
App 26531 output:     /usr/lib/ruby/2.5.0/tsort.rb:347:in `each'
App 26531 output:     /usr/lib/ruby/2.5.0/tsort.rb:347:in `call'
App 26531 output:     /usr/lib/ruby/2.5.0/tsort.rb:347:in `each_strongly_connected_component'
App 26531 output:     /usr/lib/ruby/2.5.0/tsort.rb:226:in `tsort_each'
App 26531 output:     /usr/lib/ruby/2.5.0/tsort.rb:205:in `tsort_each'
App 26531 output:     /var/lib/gems/2.5.0/gems/railties-5.2.2/lib/rails/initializable.rb:60:in `run_initializers'
App 26531 output:     /var/lib/gems/2.5.0/gems/railties-5.2.2/lib/rails/application.rb:361:in `initialize!'
App 26531 output:     /home/redmine/redmine-4.0.0/config/environment.rb:14:in `<top (required)>'
App 26531 output:     config.ru:3:in `require'
App 26531 output:     config.ru:3:in `block in <main>'
App 26531 output:     /var/lib/gems/2.5.0/gems/rack-2.0.6/lib/rack/builder.rb:55:in `instance_eval'
App 26531 output:     /var/lib/gems/2.5.0/gems/rack-2.0.6/lib/rack/builder.rb:55:in `initialize'
App 26531 output:     config.ru:1:in `new'
App 26531 output:     config.ru:1:in `<main>'
App 26531 output:     /usr/share/passenger/helper-scripts/rack-preloader.rb:101:in `eval'
App 26531 output:     /usr/share/passenger/helper-scripts/rack-preloader.rb:101:in `preload_app'
App 26531 output:     /usr/share/passenger/helper-scripts/rack-preloader.rb:189:in `block in <module:App>'
App 26531 output:     /usr/lib/ruby/vendor_ruby/phusion_passenger/loader_shared_helpers.rb:380:in `run_block_and_record_step_progress'
App 26531 output:     /usr/share/passenger/helper-scripts/rack-preloader.rb:188:in `<module:App>'
App 26531 output:     /usr/share/passenger/helper-scripts/rack-preloader.rb:30:in `<module:PhusionPassenger>'
App 26531 output:     /usr/share/passenger/helper-scripts/rack-preloader.rb:29:in `<main>'
~~~

System:
- Ubuntu 18.04.1 LTS
- Redmine 4.0
- Rails 5.2.2
- Ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-linux-gnu]

Any ideas?

**Comment by alexandermeindl** [comment]
Can you test it with my last commit in master branch, if your problem is solved with it?

**Comment by dktcoding** [comment]
Yes, `2.0.17-dev` fixes the problem!

**Comment by dktcoding** [comment]
Sorry I miss this comment,

> Thanks for reporting this!
> 
> Can you give me any information, when this error occured?
> 
>     * at a special url?
the error was present in all URLs
>     * with rake redmine:plugins:migrate
> 
>     * or with starting application server
no error was reported with `rake redmine:plugins:migrate` or restarting the app server.
> 
> 
> Do you use any other plugins?

The only other plugin I have installed is `redmine_tags 4.0.0`.

Regards

**Comment by alexandermeindl** [comment]
Thanks for your feedback. I published stable version 2.0.17 with this fix.
